scripts/build_test.py

```diff
@@ -94,6 +94,8 @@ def test_ensure_files_exist(self):
         non_existent_filepaths = [INVALID_INPUT_FILEPATH]
         # Exception will be raised at first file determined to be non-existent.
         error_message = ('File %s does not exist.') % non_existent_filepaths[0]
+        # When run on Windows, the path contains `\` which should be escaped
```

Comment:
Ah. Can you not escape/redefine the entire error message? Just escape the filepath when you are composing the message.

Reply:
For this one, I think we need to escape the whole message, as there are also special characters like `.`. But I will update the comment.